packages/wrangler/src/d1/migrations/apply.tsx

```diff
@@ -121,7 +131,7 @@ export const ApplyHandler = withConfig<BaseSqlExecuteArgs>(
 					local,
 					config,
 					database,
-					undefined,
+					isInteractive(),
```

Comment:
This, however, should definitely use the check `isInteractive() && !CI.isCI()`, I think